docs/src/main/tut/datatypes/functionk.md

```diff
@@ -121,3 +121,24 @@ type ErrorOr[A] = Either[String, A]
 val errorOrFirst: FunctionK[List, ErrorOr] =
   λ[FunctionK[List, ErrorOr]](_.headOption.toRight("ERROR: the list was empty!"))
 ```
+
+## Natural Transformation
+
+In category theory, a [Natural Transformation](https://en.wikipedia.org/wiki/Natural_transformation) provides a morphism between Functors while preserving the internal structure. It's one of the most fundamental notions of category theory.
+
+If we have two `Functor`s: `F` and `G`, then, being parametric polymorphic, `FunctionK[F, G]` is automatically a Natural Transformation between them. That is, if we have a `fk: F ~> G`, then for any combination of `A`, `B` and function `f: A => B`, the following two functions are equivalent:
+```Scala
+(fa:F[A]) => fk(F.map(fa)(f))
+```
+and
+```Scala
+(fa: F[A]) => G.map(fk(fa))(f)
+````
+
+We don't need to write a law to test the implementation of the `fk` for the above to be true. It's automatically given by its parametric polymorphism.
+
+This is why a parametric polymorphic function `FunctionK[F, G]` is sometimes referred as a Natural Transformation. However, they are two different concepts.
```

Comment:
This line could use some elaboration.. maybe say something about how `FunctionK` is more powerful than natural transformations, or natural transformations can be implemented in terms of `FunctionK` (I prefer the latter)